current_charge_range.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include "current_charge_range.hpp"
+#include <stdio.h>
+#include <string.h>
+#include <iostream>
+#include <sstream>
+
+using namespace std;
+
+int input_current_charge[] = { 1, 2, 5, 4, 10, 2, 4, 4, 2 };
```

Comment:
This input array should be part of test file and not production code.

Reply:
ok I will update. Thanks